runbotics/runbotics-orchestrator-ui/src/src-app/translations/pl/users.json

```diff
@@ -0,0 +1,16 @@
+{
+    "Users.Browse.Header.Title": "Użytkownicy",
+    "Users.Browse.Tabs.AllUsers.Label": "Lista użytkowników",
+    "Users.Browse.Tabs.AllUsers.Title": "Użytkownicy",
+    "Users.Browse.Tabs.Registration.Label": "Prośby o akceptację",
+    "Users.Browse.Tabs.Registration.Title": "Użytkownicy",
+    "Users.Register.View.Events.Success.AcceptingUser": "Sukces",
+    "Users.Register.View.Events.Error.RolesNotSelected": "Nie wybrano wszystkich wymaganych roli",
```

Comment:
I will change it in delete feature (which is waiting to be pushed) to prevent unnecessary conflicts (i changed there .json to .ts files in translations)